### ISSUE TITLE
Asset compare seems not correct

### ISSUE THREAD

**Issue by baibinghere** [Hyperion-BT/Helios#41]
```javascript
/**
 * @param {Assets} other 
 * @returns {boolean}
 */
ge(other) {
	return this.gt(other) || this.eq(other);
}
```

Base on above code, assume I have 2 assets, `this` and `other`.

`this` asset contains 5 NFTs, each NFT contains one piece:

* A(1),B(1),C(1),D(1),E(1)

`other` asset contains 1 NFT, also with one piece.

* A(1)

Then I think `this.ge(other)` should return `true`, but during my testing it returned `false`.

In my opinion, the comparing should iterate all the assets in `other` and check the same asset exists in `this` and its quantity should be larger or equal to `other`. But the implementation is not the case...

Please correct me if I misunderstand the case. Thanks a lot!

**Comment by christianschmitz** [comment]
Should indeed return `true`. I'll write a little unit test and figure out whats going on.

Note tokenNames must also be the same

**Comment by gavinharris-dev** [comment]
This could be the issue that I failed to create a replication for https://github.com/Hyperion-BT/Helios/issues/25

**Comment by christianschmitz** [comment]
Culprit was indeed poorly thought out logic.

Thank you Bing for spotting this!

Fixed as of v0.10.10

**Comment by christianschmitz** [comment]
> This could be the issue that I failed to create a replication for #25

Possibly yes. The `ge` function is used to check if additional inputs need to be added from a list of spare UTxOs (eg. when there weren't enough manually added inputs specified to cover the fees).
